assets/migrations/2019_09_15_000020_create_domains_table.php

```diff
@@ -16,8 +16,8 @@ class CreateDomainsTable extends Migration
     public function up()
     {
         Schema::create('domains', function (Blueprint $table) {
+            $table->string('domain', 255)->primary(); // don't change this
```

Comment:
Realized I have way too many of these comments in the codebase.
```suggestion
            $table->string('domain', 255)->primary();
```